spring-auto-restdocs-docs/contributing.adoc

```diff
@@ -57,6 +72,49 @@ Afterwards the test JAR is located at
 `spring-restdocs/spring-restdocs-core/build/libs/spring-restdocs-core-${spring-restdocs.version}-test.jar`
 and has to be installed with the Maven command shown in the section above.
 
+[[contributing-building-dokka-testjar]]
+==== Build Dokka Core test JAR
+
+Building the Dokka core test JAR is not required to build the project,
+but if you ever want to upgrade the version of Dokka in this project this step has to be done.
+
+We use version ${dokka.version} of Dokka in this example.
+
+Clone and checkout a specific version of Dokka:
+
+.Bash
+[source,bash]
+----
+git clone git@github.com:Kotlin/dokka.git
+cd dokka
+git checkout ${dokka.version}
+----
+
+Add the following snippted at the end of `core/build.gradle`:
+
+[source,groovy]
+----
+task testJar(type: Jar) {
+    classifier = 'tests'
+    from sourceSets.test.output
+}
+----
+
+To speed the build up and avoid test issues, once can reduce the list ob subprojects in
```

Comment:
of

Reply:
Thanks, by using a spell checker I found more :)